### PR TITLE
Feature/db rds ec2 changes

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*

*Description of changes:*

1. Added timeout for LB tf
2. Updated sleeptime and improved script to run little faster on the re-run
3. Added Godwin and Ramesh Database changes and cleaned up some the code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/backstage/dev/cm-backstage-config.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ data:
       locations:
         # Examples from a public GitHub repository.
         - type: url
-          target: https://github.com/aws-samples/appmod-blueprints/blob/main/platform/backstage/templates/catalog-info.yaml
+          target: https://github.com/aws-samples/appmod-blueprints/blob/feature/db-rds-ec2-changes/platform/backstage/templates/catalog-info.yaml
```
Comment (elamaran11) [inline packages/backstage/dev/cm-backstage-config.yaml]: this wont work even for your testing. You need absolute main HEAD URL for backstage to work.

**Review by elamaran11 — COMMENTED** [review]
See the comment

File: platform/infra/terraform/database/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+- EC2 instance with SQL Server
+- Security group for the EC2 instance
+- AWS Secrets Manager secret for storing credentials
+- SSH key pair for instance access
+
+The EC2 instance is configured with:
+- SQL Server
+- AWS CLI
+- EC2 Instance Connect
+- Java Runtime Environment (JRE)
```
Comment (elamaran11) [inline platform/infra/terraform/database/README.md:17]: Why is this required?

File: platform/infra/terraform/setup-environments.sh [hunk 3]
```diff
@@ -110,7 +113,7 @@ echo "Github Source Repo for Management EKS Cluster =" $GITHUB_URL
 cd "${REPO_ROOT}/platform/infra/terraform/"
 
 rm -rf terraform-aws-observability-accelerator/
-git clone https://github.com/aws-observability/terraform-aws-observability-accelerator.git
+git clone https://github.com/hmuthusamy/terraform-aws-observability-accelerator.git -b feature/enable-polyglot-otel-config
```
Comment (elamaran11) [inline platform/infra/terraform/setup-environments.sh:116]: Are we not replacing this to use the upstream repo. Lets use the collector config method proposed by Rodrigue and use upstream

**Review by elamaran11 — COMMENTED** [review]
@hmuthusamy This is fanstastic work. Only feedback i have is, we should not use fork. We should think about using the upstream with using collector config.

File: platform/infra/terraform/setup-environments.sh (hunk 3 above)
Comment (hmuthusamy) [reply platform/infra/terraform/setup-environments.sh:116]: We have to. But we will have to evaluate if this will work with all languages and check the configs. This fork repo has been tested out locally multiple times and opens up the OTEL config. We can use upstream repo when our config is ready and tested

Comment (elamaran11) [reply platform/infra/terraform/setup-environments.sh:116]: No my point is different. We dont need enable logic like you. All we need is a custom collector config added on top of upstream repo and then run teraform accelerator. Lets try to do that as we discussed.

**Review by elamaran11 — APPROVED** [review]
LGTM but we need to fix the Collector config and remove your fork.
